kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/CreateExternalOrderCall.java

```diff
@@ -39,29 +42,35 @@ public void run() {
         try {
             // Create external order
             openOrder = remote.createExternalOrderSync(orderJwt);
+            if (getOfferType() == OfferType.EARN) {
```

Comment:
this should be in the try catch? same as  if (getOfferType() == OfferType.SPEND) , right?